README.md

```diff
@@ -208,34 +207,6 @@ zpages:
 
 ## OpenCensus Agent
 
-### <a name="agent-building-binaries"></a>Building binaries
-
-> It is recommended that you use the latest [release](https://github.com/census-instrumentation/opencensus-service/releases).
```

Comment:
Let's perhaps keep this warning somewhere in here lest folks will resort to always building from source.

Reply:
SG, I moved it to "Usage" section.

Reply:
Great, LGTM!